test/IECoreScene/TriangulateOp.py

```diff
@@ -290,5 +291,33 @@ def testUniformIndices( self ) :
 		self.assertEqual( m2["myString"].data, m["myString"].data )
 		self.assertEqual( m2["myString"].indices, IECore.IntVectorData( [ 1, 1, 0, 0, 0, 0, 1, 1 ] ) )
 
+
+	@unittest.skipUnless( os.environ.get("CORTEX_PERFORMANCE_TEST", False), "'CORTEX_PERFORMANCE_TEST' env var not set" )
+	def testTriangulatePerformance( self ):
+
+		for i in range(10):
+			root = IECoreScene.SceneInterface.create( "/path/to/cache.scc", IECore.IndexedIO.OpenMode.Read )
```

Comment:
its a bit odd to have this test not runnable without editing the code... have we done this elsewhere?

Reply:
I would like to include a mesh which has a typical number of primvars & primitives to test the performance but this would require both sourcing such a mesh & figuring out where to put it. Any suggestions?

Perhaps I can take a production mesh as zero out all the primvars but keep the topology? This would be quite compact but I think there is a more general problem of where to locate data which we can rely on for performance tests.

Reply:
Would adding a bunch of primvars to GafferBot make it fit the required specs? I guess it'd be odd to pull that into a Cortex test though...